templates/hotgridPopupToolbar.jsx

```diff
@@ -74,7 +74,7 @@ export default function HotgridPopupToolbar(props) {
       </button>
 
       {a11yConfig._options._isPopupWrapFocusEnabled &&
-      <a className="a11y-focusguard a11y-ignore a11y-ignore-focus" tabIndex="0" role="presentation">&nbsp;</a>
+      <a className="a11y-focusguard a11y-ignore a11y-ignore-focus" tabIndex="0" role="presentation"></a>
```

Comment:
If removing the no breaking space could this be amended to
`<a className="a11y-focusguard a11y-ignore a11y-ignore-focus" tabIndex="0" role="presentation" />` for consistency with Hotgraphic popup https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/master/templates/hotgraphicPopupToolbar.jsx